lib/internal/bootstrap_node.js

```diff
@@ -5,11 +5,13 @@
 // to the performance of the startup process, many dependencies are invoked
 // lazily.
 
+/* eslint-disable no-extra-parens */
+
 'use strict';
 
-(function(process) {
+(async function(process) {
```

Comment:
Does this need a global catch handler now?

Reply:
This gets picked up in node.cc where it is actually executed (as it always has).

The catch handler is then included in the node.cc patch.